packages/mgt/tsconfig.json

```diff
@@ -4,9 +4,11 @@
     "outDir": "dist/es6",
     "sourceRoot": "src",
     "rootDir": "src",
-    "composite": true
+    "composite": true,
+    "resolveJsonModule": true,
```

Comment:
I'm wondering if these changes are actually necessary. How do they help?

Reply:
yeah I tested and unnecessary 18b93e6